### PR TITLE
Project: To Do List

### PR DESCRIPTION
**Pull request by lucasmazo32**
I hope you enjoy the project!

Features: Every changed or added items go into the first table. The reason is because this table is a FIFO storage. Therefore no order is imposed.

After refreshing the page, all added or changed Items go to their project in the correct order (by date).  This only happens once because otherwise, the changes in the DOM and the backend will be more substantial than necessary.

### CODE REVIEW

File: src/todoItem.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { getStorage, setStorage } from './localStorageData';
+
+const localData = getStorage();
+
+let idCount = localData[0];
+
+// list default value creation
+const listStored = localData[1];
+let list = {};
+if (localData[1] !== null) {
```
Comment (adriaanbd) [inline src/todoItem.js:10]: A variable has been created on line  `8` for this, please use this instead [`OPTIONAL`].

**Review by adriaanbd — CHANGES_REQUESTED** [review]
Good job overall. The code is clean and well organized. 

We have an unmet requirement:
* Users can view all to-dos in each project

A todo item should belong to a project, thus should be inside a project container [`REQUIRED`]. Right now all todos are stacked together and there is no way of knowing which todo item belongs to what project:

image

Furthermore, as you can see above, a project is being displayed like a todo item, with the same columns of name, description, priority, change, delete and due date. It is kind of confusing. The UI should be clear about what is a project and what is a todo item. It should have a distinction. Perhaps you meant to place the project name on top of all todo items belonging to that project, and that would have been clearer, but unfortunately, that is not the case here [`OPTIONAL`].

The README.md does not containa ny instruction on how to actually start a development server to serve the file. I usually expect to be able to run `npm run start`; most people would. I'd suggest inclusion of npm script `start` in your `package.json`: `"start": "webpack-dev-server --mode development --open"` [`OPTIONAL`]

Keep up the good work.

Status: `Changes requested`
Reviewer: `Adriaan Beiertz`

**Comment by lucasmazo32** [comment]
As explained in the beginning, you needed to refresh the page to see the organized version of the to do's in their project. I added a button "Organize projects" when you add or change a to-do item so the user can click it and, thus, see all the elements organized by date in their respective project container. The mentioned button will only appear if the user adds or changes at least one project, so it will not confuse more if its there.

Added the server for development mode so now you can start it with ``` npm run start:dev ```.

Thanks!

**Review by bellom — APPROVED** [review]
## Hi @lucasmazo32 @mkezmkez  :clap: 

**Great work making all the necessary corrections from the previous review.**

**NOTE: As @adriaanbd  said, it would be nice to make it more user-friendly as you know, this project could be added to your portfolio, kindly update the UI. 👍** 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**